pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -9,11 +9,64 @@
 from pdr_backend.pdr_dashboard.util.helpers import (
     get_date_period_text_for_selected_predictoors,
     select_or_clear_all_by_table,
+    check_data_loaded,
+    get_feeds_for_favourite_predictoors,
 )
 
 
 # pylint: disable=too-many-statements
 def get_callbacks_home(app):
+    @app.callback(
+        Output("feeds_table", "data"),
+        Output("feeds_table", "selected_rows"),
+        Output("predictoors_table", "data"),
+        Output("predictoors_table", "selected_rows"),
+        Output("is-initial-table-loaded", "data"),
+        Output("home_page_table_control_predictoors_table", "children"),
+        Output("home_page_table_control_feeds_table", "children"),
+        Output("table-rows-count-predictoors_table", "children"),
+        Output("table-rows-count-feeds_table", "children"),
+        [
+            Input("is-initial-data-loaded", "data"),
+        ],
+    )
+    @check_data_loaded(output_count=9)
+    def get_initial_data(_):
```

Comment:
should we also update the "stored predictoors" with what is initially selected? I remember @KatunaNorbert saying we can not load the ppss.yml favourite addresses directly into the state, but maybe with this newer implementation we can now. What do you guys think?